glide3x/h5/glide3/src/fxglide.h

```diff
@@ -2733,7 +2733,7 @@ _grSstVRetraceOn(void);
 #if USE_STANDARD_TLS_FUNC
 
 #ifdef __GNUC__
-extern
+static
```

Comment:
These should be OK.

Reply:
OK as in OK to go in, or OK as in "should work without changes"?

Reply:
The extern-to-static inline changes are OK to go in